dhcpv4/bsdp/client.go

```diff
@@ -22,6 +22,20 @@ func NewClient() *Client {
 	}
 }
 
+func castVendorOpt(ack *dhcpv4.DHCPv4) {
+	opts := ack.Options()
+	for i := 0; i < len(opts); i++ {
+		if opts[i].Code() == dhcpv4.OptionVendorSpecificInformation {
+			vendorOpt, err := ParseOptVendorSpecificInformation(opts[i].ToBytes())
+			// Oh well, we tried
+			if err != nil {
+				return
```

Comment:
returning here means that if an option decoding failed, all the subsequent options won't be tried. What about logging the error and continuing to the next?

Reply:
There will nominally be only one Vendor specific option. This is a best-effort kind of thing that's only used for pretty-printing. If it fails, oh well.

Reply:
ok, sounds good